src/components/VueHorizontalTimeline.vue

```diff
@@ -182,3 +182,13 @@ export default {
 </script>
 
 <style src="../assets/css/style.min.css" scoped></style>
+
+<style>
+.has-color-red {
```

Comment:
I like the solution, but I few like the `<style>` tag here should be added in your project and not handled by VueHorizontalTimeline.
I believe what is missing is that we should remove it from here but add somewhere in the `README.md` for people to know how to use it.
I tested it and it should work if you only add the classes in your project (with !important).
What do you think?

Reply:
Yes, I completely agree with your comment. 

If you have the time to make the changes go ahead with them, as I don't have a pc for the next couple of weeks and I can't to them right now.

Thank you very much